pkg/operator/pprof.go

```diff
@@ -15,7 +15,7 @@ func newPprofServer() *http.Server {
 	pprofMux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 
 	return &http.Server{
-		Addr:    "127.0.0.1:6060",
+		Addr:    addr,
 		Handler: pprofMux,
 	}
```

Comment:
In `start.go` it is at port 6060, which is what the go package uses also. Why change to the string var out of curiosity?

Reply:
If you read the commit message for this, it's so that we can configure different ports. It's still defaulted to 6060. We need to be able to configure what ports we listen on so that we can run more than 1 copy of reporting-operator locally, otherwise their ports would conflict. Needed for running e2e and integration at the same time locally.